### ISSUE TITLE
Integrate Google Translate into funded project pages

### ISSUE THREAD

**Issue by avicaplan** [awesomefoundation/awesomebits#258]
With an increasing number of pages being published in languages other than English, we might consider building a feature into these pages to translate the project description into a language of the user's choice. Or perhaps we should even think about running the project description through Google Translate if it isn't in the same language as the user is browsing the site, offering a link to see the original.

**Comment by jcn** [comment]
I would like to hear from chapters in countries that are not primarily English-speaking that use our internationalization. I know the Minas Gerais chapter operates (publicly) in mostly Portuguese, so I'm not sure how valuable it would be to have the ability to switch back and forth.

If someone really wants to translate, I feel like there are browser plugins that can do that anyway but this seems like a low priority.

**Comment by avicaplan** [comment]
I agree not a high priority, just a nice-to-have and perhaps not hugely complex if someone were interested in implementing. I think the main application would be for English-speaking users who click winning projects from the home page and find themselves confronted with a description in Italian, Dutch, Russian, Armenian, Portuguese, or French (the Stockholm chapter seems to post in English). A browser plug-in solves this problem, but I suspect most people don't have one set up.
